### PR TITLE
add description to "firefox" to match the other links in info.md

### PR DESCRIPTION
**Pull request by NanashiTheNameless**
## What does this PR do?

add description to "firefox" to match the other links in info.md (other languages made by translating english with deepl/google translate)

## Why is this change important?
it helps with the consistency with the about page

i saw the missing test on my fork and thought id help

## Author's checklist

please verify non-english translations as i dont speak those languages (i did my best)

### CODE REVIEW

**Comment by unixfox** [comment]
I'm not exactly sure but the translations need to be done on weblate.

**Comment by return42** [comment]
> I'm not exactly sure but the translations need to be done on weblate.

Only strings translated on weblate .. info-pages are translated here in the repo.

File: README.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. SPDX-License-Identifier: AGPL-3.0-or-later
+bout.md.. SPDX-License-Identifier: AGPL-3.0-or-later
```
Comment (return42) [inline README.rst:1]: leftover?

File: searx/infopage/de/about.md [hunk 2]
```diff
@@ -35,10 +35,10 @@ betreiben.
 SearXNG unterstützt [OpenSearch].  Weitere Informationen zum Ändern Ihrer
 Standardsuchmaschine finden Sie in der Dokumentation zu Ihrem [WEB-Browser]:
 
-- [Firefox]
+- [Firefox] - Achter de link vindt u ook enkele nuttige instructies voor Firefox.
```
Comment (return42) [inline searx/infopage/de/about.md:38]: Can't speak about other languages, but the DE translation is not DE language

File: searx/infopage/de/about.md [hunk 3]
```diff
@@ -35,10 +35,10 @@ betreiben.
...
 - [Microsoft Edge] - Hinter dem Link finden sich auch nützliche Hinweise zu
   Chrome und Safari.
-- [Chromium]-basierte Browser fügen nur Websites hinzu, zu denen der Benutzer
+- [Chromium] - basierte Browser fügen nur Websites hinzu, zu denen der Benutzer
```
Comment (return42) [inline searx/infopage/de/about.md:41]: That there is non space in "Chromium-based" was intended.

File: searx/infopage/en/about.md [hunk 4]
```diff
@@ -32,10 +32,10 @@ own server!
 SearXNG supports [OpenSearch].  For more information on changing your default
 search engine, see your browser's documentation:
 
-- [Firefox]
+- [Firefox] - Behind the link, you will also find some useful instructions for Firefox.
 - [Microsoft Edge] - Behind the link, you will also find some useful instructions
   for Chrome and Safari.
```
Comment (return42) [inline searx/infopage/en/about.md:37]: I think there is a misunderstanding: The Firefox link is only for firefox, while Edge link also has *"useful instructions for Chrome and Safari."*

**Review by return42 — COMMENTED** [review]
@CortezJEL I'm going to close this PR without merging (see my review comments) .. anyway, thanks a lot for contributing to SearXNG :+1:

File: searx/infopage/de/about.md (hunk 2 above)
Comment (NanashiTheNameless) [reply searx/infopage/de/about.md:38]: i used google translate as i dont speak dutch, my apoligies if i incorrectly did that.

File: README.rst (hunk 1 above)
Comment (NanashiTheNameless) [reply README.rst:1]: oh, thats my bad, i had y github set to do that by default, apoligies

**Pull request closed without merging**